apps/site-nft/src/components/MainNavigation/styles.module.less

```diff
@@ -12,22 +12,31 @@
 
 #mainMenu {
   grid-area: header;
-}
 
-#logoLink{
-  border-bottom: none;
-  #logo{
-      width: 150px;
-      padding-left: 10px;
-      margin-right: 10px;
-      opacity: 0.8;
-      transition: opacity .25s ease-in-out;
-      -moz-transition: opacity .25s ease-in-out;
-      -webkit-transition: opacity .25s ease-in-out;
+  .logoLink{
+    cursor: pointer;
+    height: 30px;
+    .logo{
+        transition        : opacity @pageTransitionDuration @defaultEasing;
+        -moz-transition   : opacity @pageTransitionDuration @defaultEasing;
+        -webkit-transition: opacity @pageTransitionDuration @defaultEasing;
 
-      &:hover{
-          opacity: 1;
-      }
+        &:hover {
+          opacity: 0.7;
+        }
+    }
   }
-}
 
+  .burgerMenu {
+    font-size: 16px;
```

Comment:
I -really- should be using `rem` here, right?

Reply:
Unless you want to switch everything right now. I would say it's not a big deal